### ISSUE TITLE
[BUG]  CLI does not progress from stitching step

### ISSUE THREAD

**Issue by Lukasra00** [MouseLand/cellpose#845]
**Describe the bug**
The CLI does not progress from "[INFO] stitching 89 planes using stitch_threshold=0.100 to make 3D masks". The console remains unresponsive and an ongoing process can be seen in the task manager on a high powered machine over night. 
I run Cellpose CLI with a human-in-the-loop refined model on a single-channel z-stack of shape (ZYX: 89,  7446, 11163). The file is a .tif with dtype=uint8 and memory size of 7 GB. 

**To Reproduce**
1. Launch the miniconda terminal
2. Activate the local environment, in which the cellpose local installation is (cloned from github)
3. Run cellpose using the commands seen in the run log below

**Run log**
(cellpose_io) C:\>python -m cellpose --verbose --use_gpu --pretrained_model N:\schroeder\Data\LR\cellpose_training_set\Training_round2\models\Mark4sequential_20231213_094052 --dir N:\schroeder\Data\LR\stacks\yang --savedir N:\schroeder\Data\LR\cellpose_whole\  --z_axis 0 --save_tif --stitch_threshold 0.1 --cellprob_threshold -1 --flow_threshold 0.4 --diameter 30 --no_npy
2024-01-17 11:23:00,980 [INFO] WRITING LOG OUTPUT TO C:\Users\radtkel\.cellpose\run.log
2024-01-17 11:23:00,980 [INFO]
cellpose version:       2.2.4.dev46+g5792997.d20231205
platform:               win32
python version:         3.10.13
torch version:          2.1.1+cu121
2024-01-17 11:23:01,375 [INFO] ** TORCH CUDA version installed and working. **
2024-01-17 11:23:01,375 [INFO] >>>> using GPU
2024-01-17 11:23:01,375 [INFO] >>>> running cellpose on 1 images using chan_to_seg GRAY and chan (opt) NONE
2024-01-17 11:23:01,375 [INFO] >>>> loading model N:\schroeder\Data\LR\cellpose_training_set\Training_round2\models\Mark4sequential_20231213_094052
2024-01-17 11:23:01,766 [INFO] >>>> model diam_mean =  30.000 (ROIs rescaled to this size during training)
2024-01-17 11:23:01,766 [INFO] >>>> model diam_labels =  21.753 (mean diameter of training ROIs)
2024-01-17 11:23:01,782 [INFO] >>>> using diameter 30.000 for all images
2024-01-17 11:23:01,782 [INFO] 0%|          | 0/1 [00:00<?, ?it/s]
2024-01-17 11:23:01,828 [INFO] reading tiff with 89 planes
100%|██████████████████████████████████████████████████████████████████████████████████| 89/89 [00:16<00:00,  5.27it/s]
2024-01-17 11:23:18,732 [INFO] multi-stack tiff read in as having 89 planes 1 channels█| 89/89 [00:16<00:00,  5.22it/s]
2024-01-17 11:24:19,712 [INFO]
2024-01-17 11:24:19,727 [INFO] 0%|          | 0/89 [00:00<?, ?it/s]

### (I leave out the processing of the 89 planes to not make this report too verbose)

2024-01-17 12:05:15,900 [INFO] 100%|##########| 89/89 [40:56<00:00, 27.60s/it]
2024-01-17 12:05:15,994 [INFO] network run in 2456.28s
2024-01-17 13:21:54,494 [INFO] stitching 89 planes using stitch_threshold=0.100 to make 3D masks

**Comment by carsen-stringer** [comment]
Sorry this is a known bug that will be fixed in the next release, in a week
or two

**Comment by Lukasra00** [comment]
Thank you very much for clarifying and addressing the issue in the next release!

Is there already some understanding behind the bug's mechanism that you could share? 
Then I could attempt to solve it and further harness your model for my current pipeline :)

**Comment by Lukasra00** [comment]
Is there by any chance a line in the code you could point me towards?
I could then try and fix the bug myself. Thank you in advance.

**Comment by carsen-stringer** [comment]
this is now fixed
